Sail/src/Sail/entities/systems/Gameplay/ProjectileSystem.cpp

```diff
@@ -19,8 +17,8 @@ ProjectileSystem::~ProjectileSystem() {
 
 void ProjectileSystem::update(float dt) {
 	for (auto& e : entities) {
-		PhysicsComponent* physComp = e->getComponent<PhysicsComponent>();
-		auto projectileCollisions = physComp->collisions;
+		CollisionComponent* collision = e->getComponent<CollisionComponent>();
+		auto projectileCollisions = collision->collisions;
 		for (auto& collision : projectileCollisions) {
```

Comment:
"collision" is used as variable name multiple times.

Reply:
Renamed first variable to collisionComp